.gitignore

```diff
@@ -263,4 +263,6 @@ paket-files/
 
 # Python Tools for Visual Studio (PTVS)
 __pycache__/
-*.pyc
\ No newline at end of file
+*.pyc
+
+addons/WAB
```

Comment:
why have to include this change? What's `WAB` ?

Reply:
Yes, I have renamed the addon I don't like the blizzard to catch me with this addon.

Reply:
I'm not completely sure what you trying to imply here but recently have been added a change where you have to customize the installed addon before its getting deployed into your wow installed addons folder, which means everyone has its own copy installed with a customized name(title), author, and even the in game setup command.

You are afraid of the Warden or the Blizzard client will scan your checked out repo location? This way of thinking the _anitcheat_ can scan any arbitrary folder...